project/plugins.sbt

```diff
@@ -2,4 +2,5 @@ addSbtPlugin("com.github.sbt" % "sbt-release" % "1.4.0")
 
 addSbtPlugin("org.xerial.sbt" % "sbt-sonatype" % "3.12.2")
 
-addSbtPlugin("ch.epfl.scala" % "sbt-version-policy" % "3.2.1")
\ No newline at end of file
+addSbtPlugin("ch.epfl.scala" % "sbt-version-policy" % "3.2.1")
+addDependencyTreePlugin
```

Comment:
did you mean to commit this @Divs-B ? normally you would only add it locally e.g.
```
% cat ~/.sbt/1.0/plugins/plugins.sbt    
addDependencyTreePlugin
%
```

Reply:
didn't realise it was common practice, thanks for the pointer @emdash-ie !
It certainly won't break things.

I assumed it would slow automated builds down, but the docs are not clear on the exact implication [Since it injects many tasks per subprojects, the plugin is split into two parts](https://www.scala-sbt.org/1.x/docs/sbt-1.4-Release-Notes.html#:~:text=Since%20it%20injects%20many%20tasks%20per%20subprojects%2C%20the%20plugin%20is%20split%20into%20two%20parts)